src/ape_init/_cli.py

```diff
@@ -33,6 +33,29 @@ def cli(cli_ctx, github):
             else:
                 folder.mkdir(exist_ok=False)
 
+        git_ignore_path = project_folder / ".gitignore"
+        if git_ignore_path.exists():
+            cli_ctx.logger.warning(f"'{git_ignore_path}' exists")
```

Comment:
```suggestion
            cli_ctx.logger.warning(f"Unable to create .gitignore: '{git_ignore_path}' file exists.")
```

Reply:
Changed because I am looking for a file not a directory

Reply:
Changed because I am looking for a file not a directory